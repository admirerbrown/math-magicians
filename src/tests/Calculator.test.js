import TestRenderer from 'react-test-renderer';
import CalcDesign from '../components/Calculator';

describe('Testing calcDesign component', () => {
  it('should render the calculator Ui correctly', () => {
    const page = TestRenderer.create(<CalcDesign />).toJSON();
    expect(page).toMatchSnapshot();
  });
});
