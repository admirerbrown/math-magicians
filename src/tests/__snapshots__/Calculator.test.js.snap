// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing calcDesign component should render the calculator Ui correctly 1`] = `
<div
  className="page-wrapper"
>
  <h2
    className="page-note"
  >
    Let's do some math!
  </h2>
  <div
    className="grid-container"
  >
    <div
      className="digit cal-output"
    >
      0
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      AC
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      +/-
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      %
    </div>
    <div
      className="digit orange"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      รท
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      7
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      8
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      9
    </div>
    <div
      className="digit orange"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      x
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      4
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      5
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      6
    </div>
    <div
      className="digit orange"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      -
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      1
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      2
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      3
    </div>
    <div
      className="digit orange"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      +
    </div>
    <div
      className="digit gray zero-zone"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      0
    </div>
    <div
      className="digit gray"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      .
    </div>
    <div
      className="digit orange"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      =
    </div>
  </div>
</div>
`;
